src/components/plots/styling/MultiViewEditor.jsx

```diff
@@ -74,26 +136,26 @@ const MultiViewEditor = (props) => {
   const onGeneReorder = (index, newPosition) => {
     const newPlotUuids = arrayMoveImmutable(multiViewConfig.plotUuids, index, newPosition);
 
-    onMultiViewUpdate({ plotUuids: newPlotUuids });
+    updateMultiViewWithChanges({ plotUuids: newPlotUuids });
   };
 
   const onNodeDelete = (index) => {
     const newPlotUuids = [...multiViewConfig.plotUuids];
     _.pullAt(newPlotUuids, index);
 
-    onMultiViewUpdate({ plotUuids: newPlotUuids });
+    updateMultiViewWithChanges({ plotUuids: newPlotUuids });
   };
 
   const onRowsChange = (value) => {
     setLocalNRows(value);
 
-    if (value) onMultiViewUpdate({ nrows: value });
+    if (value) updateMultiViewWithChanges({ nrows: value });
```

Comment:
nitpick(feel free to ignore): `onMultiViewChanges` sounds like a rename that would fit more with the callback syntax we seem to be using